mmaction/models/heads/bbox_head.py

```diff
@@ -22,6 +22,7 @@ class BBoxHeadAVA(nn.Module):
             'max'. Default: 'max'.
         in_channels (int): The number of input channels. Default: 2048.
         num_classes (int): The number of classes. Default: 81.
+        topk (Union[int, Tuple[int]]): k values to calculate topk accuracies. Default: (3, 5).
```

Comment:
I think the type of topk should be tuple[int], since we have `for k in topk`

Reply:
It get's passed to `multilabel_accuracy` where it's handled.
```python
    def multilabel_accuracy(self, pred, target, topk=1, thr=0.5):
        if topk is None:
            topk = ()
        elif isinstance(topk, int):
            topk = (topk, )
```
So for the result for loop, I should add the same `isinstance` handling. But since this is already in #555  I am going to close the PR anyways.